components/email/src/Email.svelte

```diff
@@ -1603,91 +1639,182 @@
             class:disable-click={activeThread &&
               activeThread.messages.length <= 0}
           >
-            <div class="from{_this.show_star ? '-star' : ''}">
-              {#if _this.show_star}
-                <div class="starred">
-                  <button
-                    id={`thread-star-${_this.thread_id}`}
-                    class={activeThread.starred ? "starred" : ""}
-                    value={_this.thread_id}
-                    role="switch"
-                    aria-checked={activeThread.starred}
-                    on:click|preventDefault={handleThreadStarClick}
-                    aria-label={`Star button for thread ${_this.thread_id}`}
-                  />
+            {#if activeThread && activeThread.messages.length <= 0}
+              {#await isThreadADraftEmail(activeThread) then isDraft}
```

Comment:
@pooja169usp why is this awaited?

Reply:
@AaronDDM there is no need here, I added it so I could get the value of `isThreadADraftEmail(activeThread)` in to a variable that can be referred in the template. 
It is just a workaround to allow for declaring variable within html.

Reply:
Wouldn't it be better to declare a new `$: isDraft = isThreadADraftEmail(activeThread)` variable? 🤔